tests/lib/cli-engine.js

```diff
@@ -598,8 +598,6 @@ describe("CLIEngine", () => {
             const report = engine.executeOnFiles(["lib/cli*.js"]);
 
             assert.equal(report.results.length, 2);
-            assert.equal(report.results[0].messages.length, 0);
```

Comment:
We cant't remove these.

Reply:
this plugin reports some warnings, so `messages.length` is not 0, which cause the CI failing.

Reply:
you can use `[assert.lengthOf](http://chaijs.com/api/assert/#method_lengthof)` function I guess.

Reply:
Personally, I would prefer to not have any warnings be reported when we lint the codebase. Would we be able to fix the warnings?

Reply:
no, it help identify potential security hotspots, but finds a lot of false positives which need triage by a human.